src/sdk/Audio/AudioStreamFormat.ts

```diff
@@ -18,6 +18,31 @@ export abstract class AudioStreamFormat {
         return AudioStreamFormatImpl.getDefaultInputFormat();
     }
 
+    /**
+     * Creates an audio context appropriate to current browser
+     * @member AudioStreamFormat.getAudioContext
+     * @function
+     * @public
+     * @returns {AudioContext} An audio context instance
```

Comment:
This should move to the Impl class to not have it published on the SDK surface.

Reply:
Done.